.rubocop.yml

```diff
@@ -0,0 +1,197 @@
+AllCops:
+  DisplayCopNames: true
+  TargetRubyVersion: 2.3
```

Comment:
There should be a matching ruby version restriction in the gemspec. Since this is a public gem, I’d say 2.2 or even as low as 2.1. Also .travis.yml should probably test every minor version we claim to be compatible with.